### PR TITLE
MM-1073 Enable youtube metadata with a google developer key

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

File: web/react/utils/utils.jsx [hunk 1]
```diff
@@ -303,18 +303,25 @@ var getYoutubeEmbed = function(link) {
     };
 
     var success = function(data) {
-        $('.video-uploader.'+youtubeId).html(data.data.uploader);
-        $('.video-title.'+youtubeId).find('a').html(data.data.title);
+        if(!data.items.length || !data.items[0].snippet) {
+          return;
+        }
+        metadata = data.items[0].snippet;
```
Comment (hmhealey) [inline web/react/utils/utils.jsx]: This line could use a var at the start so that metadata doesn't hang around for longer than it needs to.

Comment (nickago) [inline web/react/utils/utils.jsx]: Alright

**Comment by hmhealey** [comment]
K looks good now. Ready for merge.

**Comment by nickago** [comment]
Edited the config file for this

**Pull request merged**